bin/bingx_monitor.py

```
@@ -0,0 +1,226 @@
+            modified = datetime.fromtimestamp(stat.st_mtime)
+            print(f"📝 Лог файл: {size:.1f} KB, изменен {modified.strftime('%Y-%m-%d %H:%M:%S')}")
+            
+            # Последние строки лога
+            try:
+                with open(LOG_FILE, 'r', encoding='utf-8') as f:
+                    lines = f.readlines()
+                    if lines:
+                        print("📋 Последние записи:")
+                        for line in lines[-3:]:
+                            print(f"   {line.strip()}")
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)